R/rb4model.R

```diff
@@ -60,12 +60,26 @@ missing_val <- function(df, method) {
 #' ([categorical:],[numerical: ])
 #'
 #' @export
-feature_splitter<-function(x){
-  #TODO
-
+feature_splitter<-function(data){
+    # Checking if input data of the format of data frame
+    if(class(data) != 'data.frame') stop("Warning: The input data MUST be of data frame format ")
+
+    #Analysis data types of features in the data frame
+    d_types <- sapply(data, class)
```

Comment:
does `sapply` come from a package you need to add a dependency to? you can do this by typing `use_package(<package_name>)` in your console and then referring to each function using `package_name::function_name`

![image](https://user-images.githubusercontent.com/45249215/76011635-c08b1800-5ec9-11ea-9e6b-9dcee2ce85fd.png)
https://r-pkgs.org/whole-game.html#use_package

Reply:
@andrealee011, I believe `sapply` function is built-in to base R, as I couldn't find any package. I didn't import any package as well to use it for the function.

Reply:
okay !